src/elements/autocomplete-grid/autocomplete-grid-row/autocomplete-grid-row.scss

```diff
@@ -55,6 +55,9 @@
 ::slotted(sbb-autocomplete-grid-option) {
   flex: 1 1 auto;
   margin-right: calc(-1 * var(--sbb-spacing-fixed-2x));
+
+  // Used to enable ellipsis
+  overflow-x: hidden;
```

Comment:
Does this need to be hardcoded here?

Reply:
As offline discussed, it should be ok for all contexts.